addons/web_editor/static/src/scss/wysiwyg.dark.scss

```diff
@@ -15,3 +15,9 @@
 .o_link_dialog_preview {
     --o-link-dialog-preview-border: #{$border-width} solid #{$border-color};
 }
+.o_colorpicker_sections, .colorpicker .o_colorpicker_sections .o_we_color_btn {
```

Comment:
`.colorpicker .o_colorpicker_sections .o_we_color_btn` is a very specific selector, can't we reduce its specificity?